src/searchForm/SearchForm.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useContext } from 'react'
+import cc from 'classcat'
+
+import { color } from '_utils/branding'
+import SearchIcon from 'icon/searchIcon'
+import { CalendarIcon } from 'icon/calendarIcon'
+import Button, { ButtonStatus } from 'button'
+import { Bullet } from 'index'
+import { BulletTypes } from 'bullet'
+import { StandardSeat } from 'icon/standardSeat'
+import { DoubleArrowIcon } from 'icon/doubleArrowIcon'
+import Divider from 'divider'
+import { MediaSize, MediaSizeContext } from '_utils/mediaSizeProvider/MediaSizeProvider'
+
+export interface SearchFormProps {
+  className?: string
+  from?: string
+  to?: string
+  info?: string
+}
+
+export const separatorWidth: number = 14
+
+const SearchForm = ({ className }: SearchFormProps) => {
+  const mediaSize = useContext(MediaSizeContext)
+  return (
+    <div className={cc(['kirk-searchForm', className])}>
+      <div className="kirk-searchForm-from-container">
+        <div className="kirk-searchForm-from">
+          <Button status={ButtonStatus.UNSTYLED} className="kirk-searchForm-button">
+            <Bullet type={BulletTypes.SEARCH} className="kirk-bullet--searchForm" /> Leaving from
```

Comment:
There are a lot of nested CSS classes.
Maybe you could use tag selector instead of classes to simplify your code

Reply:
We normally don't use tag to target elements because we work with components which could output different html tags